doc/contributing/releases.md

```diff
@@ -827,6 +827,14 @@ or at runtime with:
 $ ./tools/release.sh -i ~/.ssh/node_id_rsa
 ```
 
+You can also specify a different ssh server address to connect to by defining
+a `RELEASE_HOST` environment variable:
```

Comment:
Would it make sense to use a prefix like `NODEJS_` for the variable name?

Reply:
I have no strong opinion, please notice this is an internal script used only by the Release Team when working on releases, not a public API.

I'm not sure if there's a standard in place for such an env variable or even if there's much in terms of prior art.

That said I'd rather not waste too much energy overthinking its name but happy to change / update the PR if someone comes up with a compelling reason to do so.

Reply:
Yes, no strong opinion either. My question is only motivated by the possible existence of an already defined `RELEASE_HOST` variable that has nothing to do with Node.js.

Reply:
good point, I've added that prefix just in case